src/components/Input/SwitchField/index.tsx

```diff
@@ -18,6 +43,7 @@ export default function SwitchField({ id, label, onChange = undefined, disabled
                         type="switch"
                         id={id}
                         disabled={disabled}
+                        checked={viewOnly.checked}
                         onChange={(e) => {
```

Comment:
if it's viewOnly, I think you can make it disabled and remove onChange

Reply:
Done